rocks-db/src/batch_client.rs

```diff
@@ -256,6 +265,33 @@ impl AssetIndexReader for Storage {
             }
         }
 
+        // compare to other data this data is saved in HashMap by token account keys, not mints
+        for token_acc in token_accounts_details.iter().flatten() {
```

Comment:
Can you get multiple accounts with the same pubkey?

Reply:
We cannot get multiple accounts with different data but same pubkey

Reply:
And we'll also not get those token accounts from other details, so how can it be in asset_index?